Author: Andreas Beckmann <anbe@debian.org>
Description: define CL_EXT_* to CL_API_* to restore support for external users of CL_EXT_* macros
 re-adds all CL_EXT_* defines that were removed in 9d46d9580cc469757ceb8d89842366f6c9a20093

--- a/CL/cl_platform.h
+++ b/CL/cl_platform.h
@@ -133,6 +133,29 @@ extern "C" {
     #define CL_API_PREFIX__VERSION_2_2_DEPRECATED CL_API_PREFIX_COMMON CL_API_PREFIX_DEPRECATED
 #endif
 
+#define CL_EXT_PREFIX_DEPRECATED CL_API_PREFIX_DEPRECATED
+#define CL_EXT_PREFIX__VERSION_1_0_DEPRECATED CL_API_PREFIX__VERSION_1_0_DEPRECATED
+#define CL_EXT_PREFIX__VERSION_1_1_DEPRECATED CL_API_PREFIX__VERSION_1_1_DEPRECATED
+#define CL_EXT_PREFIX__VERSION_1_2_DEPRECATED CL_API_PREFIX__VERSION_1_2_DEPRECATED
+#define CL_EXT_PREFIX__VERSION_2_0_DEPRECATED CL_API_PREFIX__VERSION_2_0_DEPRECATED
+#define CL_EXT_PREFIX__VERSION_2_1_DEPRECATED CL_API_PREFIX__VERSION_2_1_DEPRECATED
+#define CL_EXT_PREFIX__VERSION_2_2_DEPRECATED CL_API_PREFIX__VERSION_2_2_DEPRECATED
+#define CL_EXT_SUFFIX_DEPRECATED CL_API_SUFFIX_DEPRECATED
+#define CL_EXT_SUFFIX__EXPERIMENTAL CL_API_SUFFIX__EXPERIMENTAL
+#define CL_EXT_SUFFIX__VERSION_1_0 CL_API_SUFFIX__VERSION_1_0
+#define CL_EXT_SUFFIX__VERSION_1_0_DEPRECATED CL_API_SUFFIX__VERSION_1_0_DEPRECATED
+#define CL_EXT_SUFFIX__VERSION_1_1 CL_API_SUFFIX__VERSION_1_1
+#define CL_EXT_SUFFIX__VERSION_1_1_DEPRECATED CL_API_SUFFIX__VERSION_1_1_DEPRECATED
+#define CL_EXT_SUFFIX__VERSION_1_2 CL_API_SUFFIX__VERSION_1_2
+#define CL_EXT_SUFFIX__VERSION_1_2_DEPRECATED CL_API_SUFFIX__VERSION_1_2_DEPRECATED
+#define CL_EXT_SUFFIX__VERSION_2_0 CL_API_SUFFIX__VERSION_2_0
+#define CL_EXT_SUFFIX__VERSION_2_0_DEPRECATED CL_API_SUFFIX__VERSION_2_0_DEPRECATED
+#define CL_EXT_SUFFIX__VERSION_2_1 CL_API_SUFFIX__VERSION_2_1
+#define CL_EXT_SUFFIX__VERSION_2_1_DEPRECATED CL_API_SUFFIX__VERSION_2_1_DEPRECATED
+#define CL_EXT_SUFFIX__VERSION_2_2 CL_API_SUFFIX__VERSION_2_2
+#define CL_EXT_SUFFIX__VERSION_2_2_DEPRECATED CL_API_SUFFIX__VERSION_2_2_DEPRECATED
+#define CL_EXT_SUFFIX__VERSION_3_0 CL_API_SUFFIX__VERSION_3_0
+
 #if (defined (_WIN32) && defined(_MSC_VER))
 
 /* intptr_t is used in cl.h and provided by stddef.h in Visual C++, but not in clang */
